client/src/pages/MusicPlace/styles/FilterPage.css

```
@@ -0,0 +1,70 @@
+#form-container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: baseline;
+}
+
+#filterButton:hover {
+    background-color: #079238;
+    cursor: pointer;
+}
+
+#filter-page {
+    padding: 40px;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+#filter-page-title {
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+.select-box {
+    width: 400px;
+    height: 34px;
+    border-radius: 23px;
+    background-color: white;
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    font-size: 16px;
+    color: #707070;
+    border: 1px solid white;
+    padding-left: 5px;
+    margin-right: 30px;
+    margin-bottom: 20px;
+}
+
+.select-box:hover {
+    border: 1px solid #1DB954;
```

I suggest @ackar8985 
```suggestion
    border: 3px solid #1DB954;
```